packages/cf.js/src/app-instance.ts

```diff
@@ -1,12 +1,22 @@
 import {
   Address,
   AppABIEncodings,
+  AppAction,
   AppInstanceID,
   AppInstanceInfo,
-  BlockchainAsset
+  AppState,
+  BlockchainAsset,
+  Node
 } from "@counterfactual/common-types";
 import { BigNumber } from "ethers/utils";
 
+import { Provider } from "./provider";
+
+interface Payouts {
```

Comment:
This interface is critical -- can we discuss it on another issue or on Slack

Reply:
Sure. I can just return nothing from `uninstall()` for now so that I can merge my PR.